.circleci/config.yml

```diff
@@ -11,7 +11,53 @@ workflows:
           context: Danger
       - kitchen/lint:
           name: lint
-      # - kitchen/dokken:
-      #     name: default
-      #     suite: default
-      #     requires: [  danger, lint ]
\ No newline at end of file
+      - kitchen/dokken-single:
+          name: default-amazonlinux-2
+          suite: default
+          platform: amazonlinux-2
+          requires: [  danger, lint ]
+      - kitchen/dokken-single:
+          name: default-debian-8
+          suite: default
+          platform: debian-8
+          requires: [  danger, lint ]
+      - kitchen/dokken-single:
+          name: default-debian-9
+          suite: default
+          platform: debian-9
+          requires: [  danger, lint ]
+      - kitchen/dokken-single:
+          name: default-ubuntu-14.04
+          suite: default
+          platform: ubuntu-1404
+          requires: [  danger, lint ]
+      - kitchen/dokken-single:
+          name: default-ubuntu-16.04
+          suite: default
+          platform: ubuntu-1604
+          requires: [  danger, lint ]
+      - kitchen/dokken-single:
+          name: default-ubuntu-18.04
+          suite: default
+          platform: ubuntu-1804
+          requires: [  danger, lint ]
+      - kitchen/dokken-single:
+          name: default-centos-7
+          suite: default
+          platform: centos-7
+          requires: [  danger, lint ]
+      - kitchen/dokken-single:
+          name: default-fedora-26
+          suite: default
+          platform: fedora-26
+          requires: [  danger, lint ]
+      - kitchen/dokken-single:
+          name: default-fedora-27
+          suite: default
+          platform: fedora-27
+          requires: [  danger, lint ]
+      - kitchen/dokken-single:
```

Comment:
kitchen/dokken covers most of these platforms for less yaml :)

Reply:
It does, but it also includes centos-6, which will break the check. I don't know if there's a way to have a loop in this yaml - so platform: [ centos-7, fedora-27, ubuntu-1804 ] and didn't want to burn too many cycles on that at this point.

Also this way there's also a number of dokken jobs which can be parallelized and reported separately. Minor thing, but I've found it helpful.